### ISSUE TITLE
sfdx plugins:install fails

### ISSUE THREAD

**Issue by davidbrowaeys** [forcedotcom/cli#592]
Hi community, 

I am using Azure pipeline as a ci/cd tool and the deployment is running with ubuntu-latest vm. The installation of the sfdx-cli is successful through npm 
`npm install sfdx-cli`

however it seems the sfdx force:plugins is failing to run and return an error. 

![image](https://user-images.githubusercontent.com/20630942/92040814-6573a880-edba-11ea-9398-b38c32a8c5c3.png)

By installing a previous version of sfdx cli it seems to work though. 

Did something change with custom sfdx plugins that we need to adjust on our side ?

**Comment by azlam-abdulsalam** [comment]
We are noticing this as well, not limited to azure pipelines
and is able to reproduce in a docker image with 7.72.0

https://github.com/Accenture/sfpowerkit/issues/378

@rody @manisfdx

**Comment by rody** [comment]
Hello, I've added some [comments](https://github.com/Accenture/sfpowerkit/issues/378#issuecomment-686064440) on the issue  [Accenture/sfpowerkit#378](https://github.com/Accenture/sfpowerkit/issues/378).

## Summary

This bugs can be found in version 7.72.0 and 7.71.0 of the sfdx-cli and affects many plugin installation. Version 7.70.1 is working.

## Steps to reproduce

Attached the docker files to reproduce the issue. You check each version by building the image with the command

```sh
docker build -f Dockerfile.7.72.0 .
```
[Archive.zip](https://github.com/forcedotcom/cli/files/5165167/Archive.zip)

**Comment by davidbrowaeys** [comment]
@rody  thanks for the update. Do you have something like **stable** tag version so maybe in our pipeline we can just do `npm install sfdx-cli@stable `

**Comment by rody** [comment]
@davidbrowaeys I'm not part of forcedotcom, so I can't really help with the @stable tag for the sfdx-cli.

I think (but I'm not sure) that each published version of the sfdx-cli can be considered as stable and the beta versions are kept internal. Again I might be completely  wrong about this....

**Comment by rhobkirkcertinia** [comment]
We started hitting this issue and I tracked it back to a recent release of the npm package `@salesforce/sfdx-trust` version 3.4.2.

I would assume that the installers don't hit this issue since it's a problem with a transitive dependency, so you could try installing that way to avoid the issue.

**Comment by rody** [comment]
I tried today with the new 7.73.1 release. The issue is still present.

```docker
FROM node:14-stretch
RUN npm install -g sfdx-cli@7.73.1
RUN echo 'y' | sfdx plugins:install sfpowerkit
CMD "bash"
```

**Comment by Szandor72** [comment]
We're also hitting this issue in our CircleCI builds

**Comment by dieffrei** [comment]
Same here! Pipelines are broken.
**Workaround:** https://salesforce.stackexchange.com/questions/318537/sfdx-plugin-installation-crash-typeerror-cannot-read-property-dim-of-undefin

**Comment by azlam-abdulsalam** [comment]
@amphro  any updates on this?

**Comment by amphro** [comment]
It will be fixed this week. Sorry for the inconvenience. It is a missing dependency in the trust package.

**Comment by dieffrei** [comment]
Tks for let us know @amphro

**Comment by amphro** [comment]
Fixed in `sfdx-cli@7.74.1`

**Comment by davidbrowaeys** [comment]
@amphro  thank you seems to be fixed. But [this issue](https://github.com/forcedotcom/cli/issues/468) here seems to be blocking many deployments so can't really upgrade yet
